### PR TITLE
Write a bit of user doc for the visualizer

### PR DESCRIPTION
**Pull request by Raphalex46**

### CODE REVIEW

File: docs/Visualizer/guide.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+## The scans list page
```
Comment (fabfabBesnard) [inline docs/Visualizer/guide.md]: scan list page -> I think should scan should not be plural

**Review by fabfabBesnard — APPROVED** [review]
I made a few text changes in the guide.md
it's a good starting point for general doc !

**Comment by fabfabBesnard** [comment]
I do not know how pushing my commit into this PR... Can I let you integrate it and merge Raphaël ?

**Comment by Raphalex46** [comment]
If you pushed on the branch then it's ok. Check the commit tab on this PR to see commits associated to this PR.

**Pull request merged**